### PR TITLE
doc: hv: add comments to vIRQ APIs for documentation

### PR DESCRIPTION
**Pull request by yonghuah**
Add comments for virtual Interrupt module APIs

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
1665<br>No new violations to the coding guideline detected.

**Comment by yonghuah** [comment]
@dbkinder This patch series add comments to header and C
sources in doxygen style  for vIRQ module,  could you help to review? thanks.

**Comment by HuiHuangShi** [comment]
1665<br>No new violations to the coding guideline detected.

File: hypervisor/arch/x86/guest/vlapic.c [hunk 1]
```diff
@@ -1240,6 +1263,19 @@ vlapic_icrlo_write_handler(struct acrn_vlapic *vlapic)
 	return 0;	/* handled completely in the kernel */
 }
 
+/**
+ * @brief Get pending virtual interrupts for vLAPIC.
+ *
+ * @param vlapic[in] Pointer to target vLAPIC data structure
+ * @param vector[inout] Pointer to vector buffer and will be filled
```
Comment (gvancuts) [inline hypervisor/arch/x86/guest/vlapic.c]:
```suggestion
 * @param vecptr[inout] Pointer to vector buffer and will be filled
```

Comment (yonghuah) [reply hypervisor/arch/x86/guest/vlapic.c]: @gvancuts  thanks for comments ,will update

**Comment by HuiHuangShi** [comment]
1665<br>No new violations to the coding guideline detected.

**Comment by dbkinder** [comment]
@yonghuah I tried to use the github interface to resolve the merge conflict (in acrn.doxyfile) and looks like I messed things up.  Can you fix?

**Comment by yonghuah** [comment]
@dbkinder Sure, will rebase this PR, thanks.

**Comment by yonghuah** [comment]
@dbkinder , have updated, pls help to review, thanks.

**Comment by HuiHuangShi** [comment]
1665<br>No new violations to the coding guideline detected.

**Pull request merged**